server/access.js

```diff
@@ -1,22 +1,27 @@
 ChatterMessage.getCollection().allow({
     insert: function (userId, doc) {
         let ownsMessage = userId === doc.userId;
-        
+
         if (!ownsMessage) {
             console.log("not owner", doc);
             throw new Meteor.Error("not_owner", "You are not the owner of this message");
         }
-        
+
         let user = Meteor.users.findOne(userId);
-        
+
         if (_.isUndefined(user)) {
             console.log("user undefined");
             throw new Meteor.Error("cant_find_user", "Can't find specified user");
         }
-        
-        
-        
-        
+
         return true;
     }
-})
\ No newline at end of file
+})
+
+
+ChatterUserRoom.getCollection().allow({
```

Comment:
beware of this, better to have a method to join and do this server side to avoid duplicates, invalid data etc?